### DISCUSSION TITLE
Python3.6 - End of life advance warning

### DISCUSSION THREAD

**Discussion by gingergeeks** [PySlurm/pyslurm#217] [Announcements]
According to [Python Dev Guide](https://devguide.python.org/), [Python3.6](https://www.python.org/dev/peps/pep-0494/) is due to come to end of life on 2021-12-23. As such we plan to drop support for Python3.6 after the next PySlurm release (21.08.4).

UPDATE - As requested due to existing OS's (RHEL8 etc) still having Py3.6 as the base version even though it is EOL we will maintain build support for Py3.6.

**Comment by cjw296** [comment]
What does " drop support for Python3.6" mean here?
Actively breaking 3.6 support would be a shame given that RHEL8, and I'm sure other Linux distros, still have it as their "base version".

**Reply by gingergeeks** [reply]
@cjw296 If OS's are still trailing in the minimum Python3 version levels then we will maintain build testing for 3.6 then we will modify our stance and intent on Py3.6 support and maintain it in the build testing for sure. Is that okay ?

**Reply by cjw296** [reply]
That would be fantastic, thanks! I'm super keen to help maintain things, what's the best way to contact you?
